account_invoice_view_payment/views/account_invoice_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="invoice_form" model="ir.ui.view">
+        <field name="name">account.invoice.form</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <field name="payments_widget" position="after">
+                <field name="payment_ids" invisible="1"/>
+                <button type="object"  name="action_view_payments"
+                        class="oe_link" string="View Payments"
+                        colspan="2"
+                        attrs="{'invisible':[('payment_ids','=',[])]}"/>
```

Comment:
I would put the group `group_account_user` to every of these buttons as the payments are only visible to this group in standard odoo.

cc @jbeficent

Reply:
Done